Sources/SwiftDriver/IncrementalCompilation/ModuleDependencyGraph.swift

```diff
@@ -214,17 +200,47 @@ extension ModuleDependencyGraph {
     return nodeFinder.findNodes(for: source).map {!$0.isEmpty}
       ?? false
   }
-
-  /// Return true on success
-  func populateInputDependencySourceMap() -> Bool {
+  
+  /// Returns: false on error
+  func populateInputDependencySourceMap(
+    for purpose: InputDependencySourceMap.AdditionPurpose
+  ) -> Bool {
     let ofm = info.outputFileMap
-    let de = info.diagnosticEngine
-    return info.inputFiles.reduce(true) { okSoFar, input in
-      ofm.getDependencySource(for: input, diagnosticEngine: de)
-        .map {source in addMapEntry(input, source); return okSoFar } ?? false
+    let diags = info.diagnosticEngine
+    var allFound = true
+    for input in info.inputFiles {
+      if let source = ofm.getDependencySource(for: input, diagnosticEngine: diags) {
+        inputDependencySourceMap.addEntry(input, source, for: purpose)
+      }
+      else {
```

Comment:
Nit: OTBS here please

```suggestion
      if let source = ofm.getDependencySource(for: input, diagnosticEngine: diags) {
         inputDependencySourceMap.addEntry(input, source, for: purpose)
       } else {
```

Reply:
I did a look-up on "OTBS" and it came back with "off track horse betting"! :)

Reply:
Looks like the request is to put the else on the same line as the closing brace--no prob!